flink-table/flink-table-common/src/main/java/org/apache/flink/table/functions/BuiltInFunctionDefinitions.java

```diff
@@ -388,7 +388,16 @@ ANY, and(logical(LogicalTypeRoot.BOOLEAN), LITERAL)
                     .inputTypeStrategy(sequence(ANY))
                     .outputTypeStrategy(nullableIfArgs(explicit(DataTypes.INT().notNull())))
                     .runtimeProvided()
-                    .internal()
```

Comment:
Why did you remove the `internal` here?

Reply:
It seems that I removed it during solve conflict. I will add it back.